src/wh_nvm.c

```diff
@@ -87,6 +87,36 @@ int wh_Nvm_GetAvailable(whNvmContext* context,
             out_reclaim_size, out_reclaim_objects);
 }
 
+int wh_Nvm_AddObjectPessimistic(whNvmContext* context, whNvmMetadata *meta,
```

Comment:
Let's try a more objective name.  How about:
```suggestion
int wh_Nvm_AddObjectWithReclaim(whNvmContext* context, whNvmMetadata *meta,
```